models/rubicon/arbitrum/rubicon_arbitrum_schema.yml

```diff
@@ -0,0 +1,228 @@
+version: 2
+
+models:
+  - name: rubicon_arbitrum_trades
+    meta:
+      blockchain: arbitrum
+      project: rubicon
+      contributors: denver
+    config:
+      tags: ['arbitrum','rubicon','dex']
+    description: >
+        A table containing all trades of rubicon on arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+              - block_date
+              - blockchain
+              - project
+              - version
+              - tx_hash
+              - evt_index
+              - trace_address
+      - check_dex_seed:
```

Comment:
change this to: `check_dex_seed_legacy`
we started migrating this generic test to dunesql, so `legacy` is needed for spark

Reply:
also, since you use this test here, you should put your seed rows in `rubicon_arbitrum_trades_seed` file to `dex_trades_seed` where they are combined across projects. you could then remove that seed